scripts/fetch-python-wrapper.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# fetch-python-wrapper.sh
+# Fetches the Python wrapper file (render_jinja_template_wrapper.py) from llm-d-kv-cache-manager
+# for use in Docker builds and local development.
+#
+# This script replicates the original Dockerfile logic:
+# 1. Creates a temporary directory
+# 2. Clones the repo into that directory
+# 3. Creates the output directory structure
+# 4. Copies the wrapper file to the output location
+# 5. Cleans up the temporary directory
+
+set -euo pipefail
+
+VERSION="${1:-v0.3.2}"
```

Comment:
see earlier comment on makefile, where `v0.3.2` is hardcoded and not a build arg.